changes/989.dependencies

```diff
@@ -0,0 +1 @@
+Upgraded DeepDiff to `8.6.1` version.
```

Comment:
I think core has been marking these as `security` changes and adding the CVE number. I think we should do that here.

<img width="1731" height="355" alt="image" src="https://github.com/user-attachments/assets/c99b7517-f5c7-42cb-84fe-2524f8d725cf" />